signal_op/extract_traces.m

```diff
@@ -0,0 +1,53 @@
+function traces = extract_traces(movie_source,rec_source)
+% Extract traces from movie source and rec source. The inputs can be either
+% the paths to the sources or the sources themselves. Specifically, movie
+% can be fed as the path, 3D movie matrix or the 2D (pixels flattened)
+% movie matrix. Likewise, the rec source might be the path to the rec file
+% or the 'filters' structure in the rec file (or its transpose).
+
+regul = 0; % regularization term for LS reconstruction
```

Comment:
On second thought, shouldn't we use some sort of nonzero regularization here? My understanding is that when filters are nonorthogonal, then we have in principle multiple solutions to the LS problem, and we need to select a particular solution by additional constraints -- e.g. regularization.